.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -34,7 +34,7 @@ pipeline {
     environment {
         // Some generated env is also defined into .jenkins/dsl/jobs.groovy file
 
-        KOGITO_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
+        OPTAPLANNER_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
```

Comment:
Where do we set the `JENKINS_EMAIL_CREDS_ID` parameter; does it still point to the old value?

Reply:
it does, but will be changed to opaplanner credentials one